### PR TITLE
chore(ci): Clean reduce and update

### PR DESCRIPTION
**Pull request by Larsluph**
- Clean jobs
- Reduce CI usage by building project once for release and nightly
- Update actions

### CODE REVIEW

File: .release-please-manifest.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{".": "2.3.0"}
```
Comment (WDaan) [inline .release-please-manifest.json:1]: Will it both update the package.json and this manifest file? 🤔

I think it would only do one.

**Review by WDaan — CHANGES_REQUESTED** [review]
Looks great, but I would not change the release-please config 🤔

File: .release-please-manifest.json (hunk 1 above)
Comment (Larsluph) [reply .release-please-manifest.json:1]: Yeah the manifest is updated with the version bump, I don't really see why it is needed but without it the action step fails

**Comment by Larsluph** [comment]
> Looks great, but I would not change the release-please config 🤔

I changed it to update the action config. Parameters passed as action variables aren't specified anywhere in the docs, I assume it has been deprecated since then. I can rollback that change.

**Comment by WDaan** [comment]
All good 😄 let's see if we run into any issues

**Pull request merged**